modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractRustCodegen.java

```diff
@@ -232,6 +233,83 @@ public String sanitizeIdentifier(String name, CasingType casingType, String esca
         return name;
     }
 
+    @Override
+    public String getTypeDeclaration(Schema p) {
```

Comment:
It was error-prone because almost copy-paste between rust-generators, so only extracted from server side and unified with client .

We can see that only `client` samples were affected by this PR.